docs/concepts/what-is-a-data-converter.md

```diff
@@ -99,3 +111,17 @@ Our encryption samples use AES GCM with 256-bit keys:
 - [Go sample](https://github.com/temporalio/samples-go/tree/main/encryption)
 - [Python sample](https://github.com/temporalio/samples-python/tree/main/encryption)
 - [Java sample](https://github.com/temporalio/samples-java/tree/main/src/main/java/io/temporal/samples/encryptedpayloads)
+
+### Failure Converter
+
+A Failure Converter converts error objects to proto [Failures](/concepts/what-is-a-failure) and back.
+
+The default Failure Converter copies error messages and stack traces as plain text.
+If your errors may contain sensitive information, you can encrypt the message and stack trace by configuring the Failure Converter to use your [Payload Codec](#payload-codec), in which case it will move your `message` and `stack_trace` fields to a Payload that's run through your codec.
+
+You can make a custom Failure Converter, but doing so may make Failure serialization incompatible with other SDKs, the Web UI, and tctl.
+
+- TypeScript: [`DefaultFailureConverter`](https://typescript.temporal.io/api/classes/common.DefaultFailureConverter)
+- Go: [`GetDefaultFailureConverter`](https://pkg.go.dev/go.temporal.io/sdk@v1.18.0/temporal#GetDefaultFailureConverter)
+- Python: [`DefaultFailureConverter`](https://python.temporal.io/temporalio.converter.DefaultFailureConverter.html)
+- Java: [`FailureConverter`](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/failure/FailureConverter.html)
```

Comment:
This isn't the same `FailureConverter`, it's not customizable like it is in other SDKs, let's just mark Java as not supported at the moment.